addons/web/__manifest__.py

```diff
@@ -37,6 +37,26 @@
         #   > web.assets_common = assets common to both frontend and backend clients.
         #   > web_editor.assets_wysiwyg = assets needed by components defined in the "web_editor" module.
 
+        'web.assets_database_manager': [
```

Comment:
Mmmh ?

Reply:
I had to adapt the database manager that used the jinja dependency.

Reply:
Actually I don't need it, I am now accessing those files without using the asset manager.